firmware/application/receiver_model.cpp

```diff
@@ -1,5 +1,6 @@
 /*
  * Copyright (C) 2014 Jared Boone, ShareBrained Technology, Inc.
+ * Copyright (C) 2023 Kyle Reed
```

Comment:
Mostly reordering to match the header. I'll point out real important changes in comments.